weave-js/src/components/RadixTooltip/Tooltip.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
+
+import * as RadixTooltip from './index';
+
+export type TooltipProps = {
+  trigger: React.ReactNode;
+  content: React.ReactNode | string;
+  /**
+   * This determines if the trigger and content components should be wrapped in
+   * RadixTooltip.Trigger and RadixTooltip.Content, respectively. Otherwise, it
+   * will just assume props passed in are properly wrapped in RadixTooltip anatomy.
+   * Having this option allows a bit more flexibilty and less prop passing.
+   *
+   * Setting it to false will tell the component to not apply the RadixTooltip.Trigger
+   * and RadixTooltip.Content around the props.
+   */
+  nowrap?: boolean;
+};
+
+/**
+ * This is a more ergonomic Tooltip built on top of our RadixTooltip. If this component does not
+ * meet your needs, please refrain from modifying this and consider using RadixTooltip directly.
+ */
+export const Tooltip = ({trigger, content, nowrap = false}: TooltipProps) => {
```

Comment:
**question:** what are we gonna do about the provider? i think currently each instance configures its own provider, but i think the intent is to configure it once at the root of our application. should we go ahead and wrap both weave & app with the tooltip provider?

Reply:
yupp have a PR out for it already  https://github.com/wandb/core/pull/20803 (if you want to help review :D )